docs/user/docs/sessions/jupyter/collaboration.md

```diff
@@ -0,0 +1,62 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+# Collaboration with Jupyter Notebooks
+
+Collaborating on Jupyter notebooks is a common requirement in various
+workflows. This guide describes two main approaches to collaboration:
+individual sessions and project-level collaboration.
+
+## Collaboration in Individual Sessions
+
+If you have a notebook in your personal workspace and another user wants to
+connect to your sessions, follow these steps:
+
+### Sharing a Notebook Session
+
+1. **Request a Jupyter Session**: As usual, request a Jupyter session to start.
+2. **Connect to the Session**: Open the requested session.
+3. **Share the Session**: Click on the "Share" button on the top right of the
+   Jupyter session interface.
+4. **Include the Token**: Tick the option "Include the token in the URL".
+5. **Warning! Security Risk**: Be aware that anyone with access to the URL can
+   access your Jupyter session until termination. To revoke access, terminate
+   your session in the Capella Collaboration Manager and optionally request a
+   new session.
```

Comment:
Might clarify this point a bit.

```suggestion
   access your Jupyter session until termination. To revoke access, terminate
   your session in the Capella Collaboration Manager. After restarting it, a
   new token will be generated, and old sessions will no longer have access.
```

I found that, locally, there was a slight delay (maybe a minute or so) where the old Jupyter browser session (opened before terminating the session in collab manager) was still valid before it eventually disconnected for good. This might be worth investigating in more detail. However, I could not find a way to regain access after that disconnection, so I don't think it's a real problem.

Reply:
Jupyter itself is cached in the browser, so there might be a delay indeed. However, the backend and kernel should already be dead and cell execution etc. should result in errors like "kernel died unexpectedly". 
Did you still have access to the kernel after session termination? If yes, this might be a case where Jupyter didn't terminate after SIGTERM. Sessions have then 30 seconds to terminate properly. If Jupyter didn't do that, Kubernetes kills the Pod with SIGKILL.

Reply:
I opened a new PR for proper handling of the SIGTERM signal: https://github.com/DSD-DBS/capella-dockerimages/pull/186